app/processing/payment/calculate-scheduled-payments/get-settlement-value.js

```diff
@@ -1,5 +1,5 @@
 const getSettledValue = (previousSettlements) => {
-  return previousSettlements.reduce((total, settlement) => total + settlement.value, 0)
+  return previousSettlements.reduce((first, settlement) => Math.max(first, settlement.value), Number.NEGATIVE_INFINITY)
```

Comment:
`first` should be called `maximumValue` or sim. as it's hplding the then `maximumValue` each time